lib/plugins/aws/remove/index.js

```diff
@@ -13,14 +13,16 @@ class AwsRemove {
     this.provider = this.serverless.getProvider('aws');
 
     Object.assign(this, validate, emptyS3Bucket, removeStack, monitorStack);
+    this.removeEcrRepository = removeEcrRepository;
 
     this.hooks = {
-      'remove:remove': () =>
-        BbPromise.bind(this)
-          .then(this.validate)
-          .then(this.emptyS3Bucket)
-          .then(this.removeStack)
-          .then((cfData) => this.monitorStack('delete', cfData)),
+      'remove:remove': async () => {
+        await this.validate();
+        await this.emptyS3Bucket();
+        const cfData = await this.removeStack();
+        await this.monitorStack('delete', cfData);
+        await this.removeEcrRepository();
```

Comment:
It would be nice to construct this in a way, that we have two methods. First `checkIfEcRepositoryExist`, which we should issue at beginning but do not `await`,
and `await` after `monitorStack` succeds, and if it returns true proceed with `removeEcrRepository()`

This will ensure we do not add time to `remove` command for services where ECR is not involved

Reply:
That's a clever way to skip adding extra time for people that don't use `docker` at all, I'll update it